11LessonEleven/in-class-project/index.html

```diff
@@ -1,10 +1,40 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Intel</title>
+  <title>Lesson Eleven | In Class Project</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <section id="titlecard">
+    <h1>Order Takeaway or Delivery Food</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi vel orci in.</p>
+  </section>
+
+  <section id ="icons">
+    <div class="box">
```

Comment:
nice work with the structure, just be aware that you can select things through the id here. also, make sure the id doesn't have spaces.